packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -14,17 +14,17 @@ The only flows that support refresh tokens are the resource owner password flow
 
 > **Note:** The authorization code flow is unique in that the `offline_access` scope must be requested as part of the code request to the `/authorize` endpoint and not the request sent to the `/token` endpoint.
 
-### Get a refresh token with the code flow
+> **Note:** Whether persistent refresh token or rotating refresh token behavior is enabled depends on what type of application that you are using. When you select **Refresh Token** as an allowed grant type, [SPAs use refresh token rotation](#renew-access-and-id-tokens-with-spas) as the default behavior. Native apps and web apps use persistent refresh token behavior as the default. See [Refresh token rotation](/docs/guides/refresh-tokens/refresh-token-rotation).
```

Comment:
Can we update `state=1` value for the authorization code flow to a random UUID to keep it consistent with the other flows.
`/authorize endpoint for an authorization code and includes the offline_access scope.`